auth0_flutter/darwin/Classes/AuthAPI/AuthAPIEmailPasswordlessLoginMethodHandler.swift

```diff
@@ -0,0 +1,44 @@
+import Auth0
+
+#if os(iOS)
+import Flutter
+#else
+import FlutterMacOS
+#endif
+
+struct AuthAPIEmailPasswordlessLoginMethodHandler: MethodHandler {
+    enum Argument: String {
+        case email
+        case passwordlessType
+        case connection
```

Comment:
This param here isn't really needed here as it will always be `email`. This is something that is noted to be corrected in the next major version.